packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| action      | String                                                              | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.           |
+| requirement | [Sign On Rule Requirement Object](#sign-on-rule-requirement-object) | Specifies profile attributes to prompt user for.                                     |
+| status      | String                                                              | `ACTIVE`  or  `INACTIVE`.                                                            |
+| default     | Boolean                                                             | `true` for the first instance of this rule, which gets created by default.           |
+
+#### User Profile Rule Requirement Object
+
+| Property                   | Type   | Description                                                                                                                                                                                                                                    |
+|----------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| preRegistrationInlineHooks | String | Optional. The valid ID of an Inline Hook. Sets a Registration Inline Hook to invoke.                                                                                                                                                           |
+| profileAttributes          | Array  | Array specifying the `label` and `name` of each User Profile Attribute that should be collected from the user. `label` is human-readable text that can be displayed to the user in a web form. `name` is the name of a User Profile Attribute. |
+
+### User Profile Rule Object Example
+
+```json
+{
+	"name": "New Rule",
+    "type": "Okta:UserProfile",
+    "priority": 1,
+    "action": "ALLOW",
+    "conditions": [],
+    "requirement": {
+        "preRegistrationInlineHooks": [
+            {
+                "inlineHookId": "{{inlineHookId}}"
+            }
+        ],
+        "profileAttributes": [
+            {
+                "label": "First Name",
+                "name": "firstName",
+                "required": true
+            },
+            {
+                "label": "Last Name",
+                "name": "lastName",
+                "required": true
+            },
+            {
+                "label": "Email",
+                "name": "email",
+                "required": true
+            }
+        ]
+    }
+}
+```
+## Mappings Between Apps and Policies
+
+Mappings specify which Policies apply to which Apps. You call the `/policies/${policyId}/mappings` endpoint to set a mapping. You cannot update mappings once you have created them.
```

Comment:
nit: Suggestion; `You cannot update mappings once you have created them.` -> `You cannot update mappings once you have created them, only deletion is allowed.`